rest_server/Helper.go

```diff
@@ -0,0 +1,143 @@
+// Copyright (C) 2019-2020 Siodb GmbH. All rights reserved.
+// Use of this source code is governed by a license that can be found
+// in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func StringToByteSize(str string) (bytes uint32, err error) {
```

Comment:
Re-implemented it more properly.
Take from here 
https://github.com/siodb/misc/blob/master/tests/go_parse_number/main.go